AGPs/AGP-1H3.md

```diff
@@ -0,0 +1,55 @@
+---
+AGP: tbd
+Title: Apiary Activity Feed
+Author: @lkngtn (@1hive)
+Status: Stage II
+Track: Finance
+Created: 2019-11-12
+---
+
+# **AGP-TBD: Apiary Activity Feeds**
+
+## **Address of the transfer recipient**
+
+ `0x9213737fbb72d8f924fca55d508ae2c004413d64` Vault of the [apiary project organization](https://mainnet.aragon.org/#/apiary)
+
+## **Amount of the transfer**
+
+50,000 DAI Due upon approval
+15,000 ANT Due upon successful delivery
+
+## **Number and frequency of transfers if recurring**
+
+2 (One upon approval, one upon successful delivery).
+For the purpose of budgeting, the full amount should be debited against this quarter's budget.
+
+## **Purpose of the transfer**
+
+Due to the work on App Mining, Apiary will be indexing all transactions including "internal interactions". This gives us the ability to analyze **Activities** and we think it would be incredibly useful to be able to surface that information as a contextual activity feed. This feed would make it easier to engage and interact with organizations, making them feel much more like a social network than a company.
+
+The activity feed would be displayed in three contexts within Apiary.
+* On an Organization Detail page, showing all activities related to that organization
+* On a User's profile page, showing all activities involving them across all organizations they participate in.
+* In a User's activity feed, showing all activities across the organization's they have subscribed.   
+
+Activities will be categorized into Primary and Secondary events. Primary actions will be things like `Alice created a Vote`, whereas secondary actions will be things like `Bob voted in favor of option A`. Primary actions will exist as parent elements, and secondary actions will be nested as child elements.
+
+Additionally, we will integrate [3box Threads API](https://docs.3box.io/build/web-apps/messaging/about) to provide contextual discussions related to each Primary activity.
```

Comment:
How would this work exactly? 
 
Where will discussions take place?

Reply:
- On each piece of activity in the activity feed (a lá Facebook comments)
- On app pages (sort of like app reviews, if possible)

Reply:
Ah i see, this is going to be really cool to implement.